.github/workflows/publish-helminput-plugin-component.yaml

```diff
@@ -222,6 +223,19 @@ jobs:
               --output /tmp/plugin \
               --config "/.ocmconfig"
 
+      - name: Trigger plugin registry update
+        if: ${{ github.event_name != 'pull_request' }}
+        uses: peter-evans/repository-dispatch@ff45666b9427631e3450c54a1bcbee4d9ff4d7c0 # v3
```

Comment:
sanity check, is there nothing better for this?

Reply:
I don't know honestly. It could be tag based, but not sure where to get the component name then from and the name of the plugin which cannot be necessarily derived from a tag.